rmgpy/rmg/main.py

```diff
@@ -917,6 +924,65 @@ def makeCoreSpeciesLabelsIndependent(self):
             labels.add(potential_label)
         return oldLabels
     
+    ################################################################################
+    def processToSpeciesNetworks(self,obj):
+        """
+        breaks down the objects returned by simulate into Species and PDepNetwork
+        components
+        """
+        
+        if isinstance(obj, PDepNetwork):
+            out = [self.processPdepNetworks(obj)]
+            return out
+        elif isinstance(obj, Species):
+            return [obj]
+        elif isinstance(obj,Reaction):
+            return list(self.processReactionsToSpecies(obj))
+        elif isinstance(obj,list): #list of species
+            rspcs = self.processReactionsToSpecies([k for k in obj if isinstance(k,Reaction)])
+            spcs = {k for k in obj if isinstance(k,Species)} | rspcs
+            nworks,pspcs = self.processPdepNetworks([k for k in obj if isinstance(k,PDepNetwork)])
+            spcs = list(spcs-pspcs) #avoid duplicate species
+            return spcs+nworks
+        else:
+            raise TypeError("improper call, obj input was incorrect")
+
+    def processPdepNetworks(self,obj):
+        """
+        properly processes PDepNetwork objects and lists of PDepNetwork objects returned from simulate
+        """
+        reactionSystem = self.reactionSystem
```

Comment:
RMG-tests is failing because there is no `self.reactionSystem` attribute (only `self.reactionSystems`). I think you still need to pass in `reactionSystem` to these methods.